source/data-formats/extended-json.txt

```diff
@@ -0,0 +1,203 @@
+.. _node-extended-json:
+
+============================
+Work with Extended JSON Data
+============================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: code examples, bson, relaxed, canonical, legacy
+
+Overview
+--------
+
+JSON is a data format that represents the values of objects, arrays, numbers,
+strings, booleans, and nulls. The **Extended JSON** format defines a reserved
+set of keys prefixed with "``$``" to represent field type information that
+directly corresponds to each type in BSON, the format that MongoDB uses to
+store data.
+
+Extended JSON Formats
+---------------------
+
+MongoDB Extended JSON features different string formats to represent BSON data.
+Each of the different formats conform to the JSON RFC
+and meet specific use cases. The **Extended** format, also known as the
+**Canonical** format, features specific representations for every BSON type
+for bidirectional conversion without loss of information. The **Relaxed Mode**
+format is more concise and closer to ordinary JSON, but does not represent
+all the type information such as the specific byte size of number fields.
```

Comment:
```suggestion
all the type information such as the specific bit width of numeric fields.
```

Because I believe what you are referencing is $numberLong, $numberInt being represented as plain numbers in relaxed mode. Size and number magnitude are easy to confuse so the phrase bit width is used to be specific about the amount of precision